creds/catalog.json

```diff
@@ -0,0 +1,38 @@
+{
+  "streams": [
```

Comment:
why are these files in git PR? once check the PR after raising it

Reply:
whoever debugging or making changes next time will not have to create the files again so that is why saved 
also added in .gitignore so that not over written again